lib/ezcater_rubocop/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module EzcaterRubocop
-  VERSION = "6.1.2"
+  VERSION = "6.2.0"
```

Comment:
I changed to 7.0.0 in [8b91803](https://github.com/ezcater/ezcater_rubocop/pull/130/commits/8b918034e405c8eee9ce46572758896c09d98815), the only reason I picked minor bump is because that is what we did [last time](https://github.com/ezcater/ezcater_rubocop/pull/96/files), so I do not have strong opinions about this.